packages/@angular/cli/models/webpack-configs/common.ts

```diff
@@ -58,6 +61,18 @@ export function getCommonConfig(wco: WebpackConfigOptions) {
       let scriptPath = `script-loader!${script.path}`;
       if (script.lazy) { lazyChunks.push(script.entry); }
       entryPoints[script.entry] = (entryPoints[script.entry] || []).concat(scriptPath);
+      inlineChunks.push(script.entry);
+    });
+  }
+
+  // process static libraries
+  if (appConfig.libs.length > 0) {
```

Comment:
Alternatively, we could extend the `ExtraEntry` interface and eliminate the `apps[0].libs` configuration option.

We could require scripts to use the static option, in order to skip the webpack script-loader module and to bypass the inline chunk registration, as shown in the commit

```ts
export interface ExtraEntry {
  ...
  static?: boolean;
}
```

On the other hand, we could require scripts to be registered explicitly to use the global scope

```ts
export interface ExtraEntry {
  ...
  global?: boolean;
}
```

and compile all scripts without code splitting by default. In both cases, the lazy option would remain optional. However, the last suggestion would be a breaking change.

@hansl @filipesilva